### PR TITLE
feat: update donate page UI

### PR DESCRIPTION
**Pull request by ahmaxed**
### Donate page's three sections:
- Call to action
- Contexual details
- FAQ

### Call to action:
The Call to action will have two background variations based on the selected theme:

both

The call to action have two variations based on the donation status:
——— non-donors
Help Our Charity Do More

freeCodeCamp is a highly efficient education charity.
When you donate to freeCodeCamp, you help people learn new skills and provide for their families.
You also help us create new resources for you to use to expand your own technology skills.

———— donors
Thank you for being a supporter.

Your contribution will be crucial in creating resources that empower millions to learn new skills and support their families.
Want to make a bigger one-time donation, mail us a check, or give in other ways? Here are many [other ways you can support our charity's mission](https://www.freecodecamp.org/news/how-to-donate-to-free-code-camp).
If you want to make an additional monthly donation, please proceed with selecting a monthly amount from the donation form.

### Contextual Details:
This section have 3 lists and a final call to action. The copy is as following:

Benefits from becoming a Supporter:

- No more donation popups
- You'll get a Supporter badge
- Your profile image will get a golden halo around it
- You'll gain access to special Supporter Discord channels
- And more benefits to come in 2024

Current initiatives:

- Creating new Javascript and Python curriculula
- Creating English and math curriculula
- Translating our curriculum and tutorials to 32 languages
- Creating a free accredited computer science degree

Our Community Achievements This Year:

- Published 114 full-length courses on YouTube
- Published 1,045 text-based coding tutorials and 20 free books through freeCodeCamp Press
- Merged 2,753 code contributions into our open source repositories on GitHub
- Translated 2,106,203 words to make our curriculum and tutorials more accessible to speakers of many world languages

As you can see, we're getting things done. So you can rest assured that we'll put your donations to good use.
Get the benefits and the knowledge that you’re helping our charity change education for the better. Become a Supporter today.

Note: The contextual details remains almost the same for donors and non-donors. Only the last sentence is not displayed when a user is donating.

FAQ:
The FAQ section has not been changed from the previous version.

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by ahmaxed** [comment]
Once the copy has been approved by Quincy, I will finalize the pr accordingly.

File: client/src/components/Donation/donation-text-components.tsx [hunk 1]
```diff
@@ -167,6 +190,87 @@ export const DonationFaqText = (): JSX.Element => {
   ];
 
   return (
-    <>{faqItems.map((item, iterator) => FaqItem(item.Q, item.A, iterator))}</>
+    <>
+      <h2 data-playwright-test-label='faq-head'>{t('donate.faq')}:</h2>
```
Comment (ahmaxed) [inline client/src/components/Donation/donation-text-components.tsx:194]:
```suggestion
      <h2 data-playwright-test-label='faq-head'>{t('donate.faq')}</h2>
```

File: client/i18n/locales/english/translations.json [hunk 2]
```diff
@@ -557,7 +557,27 @@
...
+    "crucial-contribution": "Your contribution will be crucial in creating resources that empower millions to learn new skills and support their families.",
+    "if-another-monthly": "If you want to make another monthly donation, please proceed with selecting your monthly donation amount.",
+    "support-benefits-title": "Benefits from becoming a Supporter:",
+    "support-benefits-1": "No more donation popups",
+    "support-benefits-2": "You'll get a Supporter badge",
+    "support-benefits-3": "Your profile image will get a golden halo around it",
+    "support-benefits-4": "You'll gain access to special Supporter Discord channels",
+    "support-benefits-5": "And more benefits to come in 2024",
+    "current-initiatives-title": "Current Initiatives:",
+    "current-initiatives-1": "Creating new Javascript and Python curriculula",
```
Comment (ahmaxed) [inline client/i18n/locales/english/translations.json:570]:
```suggestion
    "current-initiatives-1": "Creating new Javascript and Python curricula",
```

File: client/i18n/locales/english/translations.json [hunk 3]
```diff
@@ -557,7 +557,27 @@
...
+    "current-initiatives-2": "Creating English and math curriculula",
```
Comment (ahmaxed) [inline client/i18n/locales/english/translations.json:571]:
```suggestion
    "current-initiatives-2": "Creating English and math curricula",
```

File: client/i18n/locales/english/translations.json [hunk 4]
```diff
@@ -557,7 +557,27 @@
...
+    "current-initiatives-3": "Translating our curriculum and tutorials to 32 languages",
+    "current-initiatives-4": "Creating a free accredited computer science degree",
```
Comment (ahmaxed) [inline client/i18n/locales/english/translations.json:573]:
```suggestion
    "current-initiatives-4": "Creating a free accredited computer science bachelor's degree",
```

File: client/i18n/locales/english/translations.json [hunk 5]
```diff
@@ -557,7 +557,27 @@
...
+    "support-benefits-title": "Benefits from becoming a Supporter:",
+    "support-benefits-1": "No more donation popups",
+    "support-benefits-2": "You'll get a Supporter badge",
+    "support-benefits-3": "Your profile image will get a golden halo around it",
+    "support-benefits-4": "You'll gain access to special Supporter Discord channels",
+    "support-benefits-5": "And more benefits to come in 2024",
+    "current-initiatives-title": "Current Initiatives:",
+    "current-initiatives-1": "Creating new Javascript and Python curriculula",
+    "current-initiatives-2": "Creating English and math curriculula",
+    "current-initiatives-3": "Translating our curriculum and tutorials to 32 languages",
```
Comment (ahmaxed) [inline client/i18n/locales/english/translations.json:572]:
```suggestion
    "current-initiatives-3": "Translating our curriculum and tutorials into 32 languages",
```

File: client/i18n/locales/english/translations.json [hunk 6]
```diff
@@ -557,7 +557,27 @@
     "anything-else": "Is there anything else I can learn about donating to freeCodeCamp.org?",
     "other-support": "If there is some other way you'd like to support our charity and its mission that isn't listed here, or if you have any questions at all, please email Quincy at quincy@freecodecamp.org.",
     "bear-progress-alt": "Illustration of an adorable teddy bear with a pleading expression holding an empty money jar.",
-    "bear-completion-alt": "Illustration of an adorable teddy bear holding a large trophy."
+    "bear-completion-alt": "Illustration of an adorable teddy bear holding a large trophy.",
+    "crucial-contribution": "Your contribution will be crucial in creating resources that empower millions to learn new skills and support their families.",
+    "if-another-monthly": "If you want to make another monthly donation, please proceed with selecting your monthly donation amount.",
+    "support-benefits-title": "Benefits from becoming a Supporter:",
+    "support-benefits-1": "No more donation popups",
```
Comment (ahmaxed) [inline client/i18n/locales/english/translations.json:564]:
```suggestion
    "support-benefits-1": "No more donation prompt popups",
```

File: client/i18n/locales/english/translations.json [hunk 7]
```diff
@@ -557,7 +557,27 @@
     "anything-else": "Is there anything else I can learn about donating to freeCodeCamp.org?",
     "other-support": "If there is some other way you'd like to support our charity and its mission that isn't listed here, or if you have any questions at all, please email Quincy at quincy@freecodecamp.org.",
     "bear-progress-alt": "Illustration of an adorable teddy bear with a pleading expression holding an empty money jar.",
-    "bear-completion-alt": "Illustration of an adorable teddy bear holding a large trophy."
+    "bear-completion-alt": "Illustration of an adorable teddy bear holding a large trophy.",
+    "crucial-contribution": "Your contribution will be crucial in creating resources that empower millions to learn new skills and support their families.",
```
Comment (ahmaxed) [inline client/i18n/locales/english/translations.json:561]:
```suggestion
    "crucial-contribution": "Your contribution will be crucial in creating resources that empower millions of people to learn new skills and support their families.",
```

File: client/src/components/Donation/donation-text-components.tsx [hunk 8]
```diff
@@ -14,10 +15,12 @@ const POBOX = (
   </>
 );
 
-export const DonationText = (): JSX.Element => {
+export const CtaText = (): JSX.Element => {
   const { t } = useTranslation();
   return (
     <>
+      <h1 data-playwright-test-label='main-head'>{t('donate.help-more')}</h1>
```
[the remark this answers is not in the document]
Comment (ahmaxed) [reply client/src/components/Donation/donation-text-components.tsx:22]: That is a pretty good consideration. 
I will do the research and update the donate page in the next iteration.

File: client/i18n/locales/english/translations.json [hunk 9]
```diff
@@ -565,7 +565,27 @@
...
+    "support-benefits-5": "And more benefits to come in 2024",
+    "current-initiatives-title": "Current Initiatives:",
+    "current-initiatives-1": "Creating new Javascript and Python curricula",
+    "current-initiatives-2": "Creating English and math curricula",
+    "current-initiatives-3": "Translating our curriculum and tutorials into 32 languages",
+    "current-initiatives-4": "Creating a free accredited computer science bachelor's degree",
+    "community-achivements-title": "Our Community Achievements This Year:",
+    "community-achivements-1": "Published <0>114</0> full-length courses on YouTube.",
+    "community-achivements-2": "Published <0>1,045</0> text-based coding tutorials and <0>20</0> free books through freeCodeCamp Press.",
+    "community-achivements-3": "Merged <0>2,753</0> code contributions into our open sourcerepositories on GitHub",
```
Comment (ahmaxed) [inline client/i18n/locales/english/translations.json:585]:
```suggestion
    "community-achivements-3": "Merged <0>2,753</0> code contributions into our open source repositories on GitHub",
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
This and the tests need updating. Everything else LGTM 👍

File: client/i18n/locales/english/translations.json (hunk 9 above)
[the remark this answers is not in the document]
Comment (ahmaxed) [reply client/i18n/locales/english/translations.json:585]: Thanks for catching that. I will remove all periods so it is consistent with the other lists.

File: client/i18n/locales/english/translations.json [hunk 10]
```diff
@@ -565,7 +565,27 @@
...
+    "community-achivements-4": "Translated <0>2,106,203</0> words to make our curriculum and tutorials more accessible to speakers of many world languages",
```
Comment (ahmaxed) [inline client/i18n/locales/english/translations.json:586]:
```suggestion
    "community-achivements-1": "Published <0>114</0> full-length courses on YouTube",
    "community-achivements-2": "Published <0>1,045</0> text-based coding tutorials and <0>20</0> free books through freeCodeCamp Press",
    "community-achivements-3": "Merged <0>2,753</0> code contributions into our open source repositories on GitHub",
    "community-achivements-4": "Translated <0>2,106,203</0> words to make our curriculum and tutorials more accessible to speakers of many world languages",
```

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Review by huyenltnguyen — APPROVED** [review]

**Pull request merged**